cli/injector/main.go

```diff
@@ -50,7 +55,8 @@ var (
 	ms                   metrics.Sink
 	sink                 string
 	level                string
-	targetContainerIDs   []string
+	tmpTargetContainers  []string
```

Comment:
Would it be accurate to call this `targetContainerNames`?

Reply:
Oh, no, it's a raw string holding both names and ids?

Reply:
Ah yeah, sorry I was wrong, that's name:id, not sure if it's ideal. I could definitely only transmit the names and get the ids in the injector. What do you think?

Reply:
Oh no, I think the format is great, maybe we could call it `rawTargetContainers` or `initialTargetContainers`? `tmp` was confusing to me, even though they are technically temporary (until/if we need to re-inject)